libs/react/src/api/useApiClient.ts

```
@@ -0,0 +1,87 @@
+
+      cleanups.push(
+        dynamicRegistry.registerTool({
+          name: toolName,
+          description: op.description,
+          inputSchema: op.inputSchema,
+          execute,
+        }),
+      );
+    }
+
+    return () => cleanups.forEach((fn) => fn());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)